### PR TITLE
4K standardization

### PR DESCRIPTION
**Pull request by tomkordic**
According to this doc: https://docs.mux.com/guides/minimize-processing-time#standard-input-specs and @andrewjl-mux Release branch: https://github.com/muxinc/swift-upload-sdk/tree/releases/v1.0.0

### CODE REVIEW

File: library/src/main/java/com/mux/video/upload/internal/options/DirectUploadOptions.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.mux.video.upload.internal.options
+
+class DirectUploadOptions {
```
Comment (daytime-em) [inline library/src/main/java/com/mux/video/upload/internal/options/DirectUploadOptions.kt:3]: If this isn't used, can we delete it?

**Review by daytime-em — CHANGES_REQUESTED** [review]
Looks great, just had one request!

File: library/src/main/java/com/mux/video/upload/internal/options/DirectUploadOptions.kt (hunk 1 above)
Comment (tomkordic) [reply library/src/main/java/com/mux/video/upload/internal/options/DirectUploadOptions.kt:3]: Sorry, yes, this is sufficient

File: library/src/main/java/com/mux/video/upload/api/MuxUpload.kt [hunk 2]
```diff
@@ -344,13 +345,23 @@ class MuxUpload private constructor(
       return this
     }
 
+    /**
+     * If requested, the Upload SDK will try to standardize the input file in order to optimize it
+     * for use with Mux Video
+     */
+    @Suppress("unused")
+    fun standardizationRequested(enabled: Boolean, maxResolution: MaximumResolution): Builder {
```
Comment (andrewjl-mux) [inline library/src/main/java/com/mux/video/upload/api/MuxUpload.kt:353]: Should the parameter name be updated to `requested` here?

File: library/src/main/java/com/mux/video/upload/api/MuxUpload.kt [hunk 3]
```diff
@@ -344,13 +345,23 @@ class MuxUpload private constructor(
...
+    }
+
     /**
      * If requested, the Upload SDK will try to standardize the input file in order to optimize it
      * for use with Mux Video
      */
     @Suppress("unused")
     fun standardizationRequested(enabled: Boolean): Builder {
-      uploadInfo.update(standardizationRequested = enabled)
+      uploadInfo.update(InputStandardization(enabled))
```
Comment (andrewjl-mux) [inline library/src/main/java/com/mux/video/upload/api/MuxUpload.kt:364]: And here as well, should this be updated to `requested`?

File: library/src/main/java/com/mux/video/upload/api/MuxUpload.kt (hunk 2 above)
Comment (daytime-em) [reply library/src/main/java/com/mux/video/upload/api/MuxUpload.kt:353]: Will add in a separate PR, so we can get this merged.

**Comment by daytime-em** [comment]
Thanks for the changes, tom!!

**Pull request merged**